metrics/main.py

```diff
@@ -267,6 +269,8 @@ def run_analysis(
                 os.makedirs(home_path + f"/disk/Eye-tracking/Results/{subject_name}")
             if not os.path.exists(home_path + f"/disk/Eye-tracking/Results/{subject_name}/{move_surname}"):
                 os.makedirs(home_path + f"/disk/Eye-tracking/Results/{subject_name}/{move_surname}")
+            if not os.path.exists(home_path + f"/Documents/StageMathieu/DataTrampo/Xsens_pkl/{subject_name}/{move_surname}"):
```

Comment:
please create a variable for the filepath, then create the specific subpaths using this variable